jina/types/request/control.py

```diff
@@ -6,4 +8,14 @@
 class ControlRequest(Request, CommandMixin):
     """Control request class."""
 
-    pass
+    @property
+    def args(self):
+        """struct args
+
+
+        .. #noqa: DAR201"""
+        return self.proto._args
+
+    @args.setter
+    def args(self, value: Dict):
+        self.proto.args = value
```

Comment:
I thought this should be `self.proto.args.update(value)`!? If we call the update outside, I'd rather implement a 
```
def update_args(self, value: Dict):
    self.proto.args.update(value)
```
to not manipulate state outside of this object.

Reply:
Fixed